### ISSUE TITLE
Configure extension to use absolute path

### ISSUE THREAD

**Issue by brizaldi** [luanpotter/vscode-dart-import#39]
Hi, just wondering, is there a way to customize this extension so that it use absolute path instead of relative path?

**Comment by IgorKrupenja** [comment]
If I get the issue description right, this is about changing relative imports to package imports — opposite of what this extension does now. @luanpotter Would you be willing to accept a PR here, adding a separate command? If yes, I am willing to attempt a PR here 🙂

**Comment by luanpotter** [comment]
@igor-krupenja as long as it's a separate option, so users can choose their preferred standard, that would be awesome! lmk if I can help with anything :)

**Comment by IgorKrupenja** [comment]
Yep, the idea was to create a separate option. I will start looking into this mostly likely towards the end of next week. I will let you know then if I need any help 👍

**Comment by doppio** [comment]
@igor-krupenja Have you had a chance to take a look at this? I'm interested in the same feature. 🙂

**Comment by kuyazee** [comment]
any update for this

**Comment by mapacheverdugo** [comment]
Check the new #46 PR
